unpub/lib/src/static/main.dart.js.dart

```diff
@@ -13728,7 +13728,7 @@ p=G.cf(H.d(a9.V(C.h,q),"\$iaT"),H.d(a9.V(C.i,q),"\$iaW"),a1,a0.dx)
 a0.f=new G.bA(p)
 o=T.B(a8,a0.dx,"img")
 T.an(o,"alt","dart pub logo")
-T.an(o,"src","https://pub.dev/static/img/pub-dev-logo-2x.png?hash=umitaheu8hl7gd3mineshk2koqfngugi")
+T.an(o,"src","https://pub.flutter-io.cn/static/img/pub-dev-logo.svg?hash=40fqenbgtbjcekk60vd5dg5mr22bv99t")
```

Comment:
Why is the logo from `pub.flutter-io.cn` needed here?

Reply:
This is a link to the corresponding icon found in "pub. dev"

Reply:
Yea but why not use `https://pub.dev/static/img/pub-dev-logo.svg?hash=40fqenbgtbjcekk60vd5dg5mr22bv99t` directly?